apps/remote_control/lib/lexical/remote_control/build/document/compilers/eex.ex

```diff
@@ -111,6 +112,20 @@ defmodule Lexical.RemoteControl.Build.Document.Compilers.EEx do
     end)
   end
 
+  defp reject_undefined_variables(errors) do
+    # NOTE: Ignoring error for undefined variables
+    # for example:
+    # ```heex
+    # <.inputs_for :let={finner} field={f[:inner]}>
+    #  <%= finner %>
+    # </.inputs_for>
+    # ```
+    # the `finner` variable is defined by `:let` of `inputs_for`
+    Enum.reject(errors, fn %Result{message: message} ->
```

Comment:
where do the functions come from? Is there a way we can either include them into the heex template or ignore the functions that are default imported?

Reply:
yeah, I reverted this, and tried another approach.

Reply:
awesome